### DISCUSSION TITLE
code fences do not work with Stringify.

### DISCUSSION THREAD

**Discussion by kangruixiang** [remarkjs/remark#545] [Q&A] [answered]
Hello. Hope you are doing well. I have the following code block like this:

\`\`\`
code
\`\`\`

My processor looks like this: 

```
const unified = require("unified");
const remarkParse = require("remark-parse");
const remarkStringify = require("remark-stringify");

const processor = unified()
  .use(remarkParse, { commonmark: true, gfm: false })
  .use(remarkStringify, {
    commonmark: true,
    gfm: false,
    fences: true,
  })

newcontent = processor.stringify(oldcontent)
```

Despite having fences option turned on, I get the following output:

\`
code
\`

Any thoughts? Thank you.

**Comment by ChristianMurphy** [comment, marked as answer]
@kangruixiang what version of `unified`, `remark-parse`, and `remark-stringify` are you using?
Could you share a codesandbox or runkit with a runnable example of the issue?
https://runkit.com/christianmurphy/5f8d1a70bf1adb001a70e767 is a starter for creating a runkit.

**Reply by wooorm** [reply]
@kangruixiang also, can you use proper fenced or indented code to show your input, actual output, and expected output?

~~~markdown
```js
code()
```

````markdown
use more ticks if you have ticks in here
````

    indented code
~~~

**Reply by kangruixiang** [reply]
Ok give me some time. I think I may be figuring it out with the latest updated version.

**Reply by kangruixiang** [reply]
Ok updating remark 12 to 13 and remark-parse 8 to 9 fixed the issue. But it also broke remark-wiki-link. It seems like you have already started a separate issue with landakram. Hopefully that gets fixed.

**Reply by wooorm** [reply]
Could you update this post with what was broken, and what fixed it, so that other people running into this problem can find what solves it?

**Reply by kangruixiang** [reply]
Ok in remark 12 and remark-parse 8, I have following code:
```` 
```
code
```
````

This is what I have as export after using aforementioned processor:
```
 `
 code
 `
```

When I updated to remark 13 and remark-parse 9, this was fixed.
